data/hai/hai reveal 2 start.txt

```diff
@@ -258,7 +258,7 @@ mission "Hai Rescue: Ooonem"
 				`	"Sorry to bother you. I'll leave you to your business."`
 					goto decline
 			label hai
-			`	"I was going to meet a businessman on Shroud to arrange a large-scale import deal for my research project. I asked a human merchant to take me there, but I was tricked. The businessman doesn't really exist; the deal was faked. The merchant locked me in a cell and brought me here instead. When he opened the cell, I managed to grab his rifle, shoot him, and hide. It's amazing how slowly humans move."`
+			`	"I was going to meet a businessman on Shroud to arrange a large-scale import deal for my research project. I asked a human merchant to take me there, but I was tricked. The businessman doesn't really exist and the deal was faked. The merchant locked me in a cell and brought me here instead. When he opened the cell, I managed to grab his rifle, shoot him, and hide. It's amazing how slowly humans move."`
```

Comment:
It seemed to make more sense that she would keep talking, instead of pausing to say that the deal was faked. But, I suppose that if she was speaking fast, she would have paused. I will add the comma anyway.